src/www/containers/settingspage/SettingsPage.jsx

```diff
@@ -0,0 +1,438 @@
+          } else {
+            console.log("Fail submitPassword");
+          }
+        };
+
+        updatePicture = (profilePicture, target) => {
+          axios
+            .put("api/users/me/photo", {
+              profilePicture,
+            })
+            .then((res) => this.handleResponse(res.data, target))
+            .catch((err) => this.handleResponse(err.response.data, target));
+        };
+
+        removePicture = async (event) => {
+          event.preventDefault();
+          this.updatePicture("", event.target);
+          console.log("Fail removePicture");
+        };
+
+        submitPicture = async (event) => {
+          event.preventDefault();
+          const { profilePicture } = this.state;
+          if (this.state.profilePicture !== "") {
+            console.log(profilePicture);
+            this.updatePicture(profilePicture, event.target);
+          } else {
+            console.log("Fail submitPicture");
+          }
+        };
+
+        render() {
+          //console.log(this.props);
+          if (!this.props.loggedIn) {
+            return null; //<Redirect to="login" />;
+          }
+
+          const { t, user } = this.props;
+          const { username } = user;
+          //const { username, profilePhoto } = user;
+          //const photo = profilePhoto || "/img/fakedata/profilePhoto.png";
+          //const { selectedOption } = this.state;
+
+          return (
+            <div className="SettingsPage">
+              <div className="SettingsPage_navigation">
+                <div className="SettingsPage_navigation_header">
+                  <h2>{username}</h2>
+                  <h5>{t("settings.personal_settings")}</h5>
+                </div>
+                <div className="SettingsPage_navigation_tabs">
+                  <div
+                    className={
+                      this.state.activeTab === "profile" ? "active" : ""
+                    }
+                    onClick={() => this.changeTabs("profile")}
+                  >
+                    {t("settings.profile")}
+                  </div>
+                  <div
+                    className={
+                      this.state.activeTab === "security" ? "active" : ""
+                    }
+                    onClick={() => this.changeTabs("security")}
+                  >
+                    {t("settings.security")}
+                  </div>
+                </div>
+              </div>
+              <div className="SettingsPage_content">
+                <div
+                  className={
+                    this.state.activeTab === "profile" ? " active" : ""
+                  }
+                >
+                  <div className="first_panel">
+                    <h2 className="header">{t("settings.profile")}</h2>
+                    <form
+                      id="profile-form"
+                      method="post"
+                      action="/settings/submit"
```

Comment:
Is this really a valid action URL? We're using multiple as I see per page, so I think we can skip this one.